docs/standard/frameworks.md

```diff
@@ -14,54 +14,31 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
 
 # Target frameworks
 
-*Frameworks* define the objects, methods, and tools that you use to create apps and libraries. The .NET Framework is used to create apps and libraries primarily for execution on systems running a Windows operating system. .NET Core includes a framework that allows you to build apps and libraries that run on a variety of operating systems.
+Application Programming Interfaces (APIs) are contracts between software systems describing how their objects and methods can interact. Frameworks are made up of collections of APIs. When you target one or more frameworks in an app or library, you're specifying the sets and versions of APIs you'd like the app or library to use.
 
-The terms *framework* and *platform* are sometimes confusing given how they're used in phrases and their context. For example, you'll see .NET Core described as a framework in the context of building apps and libraries and also described as a platform in the context of where app and library code is executed. A *computing platform* describes *where and how* an application is run. Since .NET Core executes code with the [.NET Core Common Language Runtime (CoreCLR)](https://github.com/dotnet/coreclr), it's also a platform. The same is true of the .NET Framework, which has the [Common Language Runtime (CLR)](clr.md) to execute app and library code that was developed with the .NET Framework's framework objects, methods, and tools.
+An app or library can target a version of [.NET Standard](~/docs/standard/net-standard.md). .NET Standard versions aren't frameworks but represent standardized sets of APIs across all .NET frameworks. For example, a library can target .NET Standard 1.6 and gain access to APIs that function across several .NET implementations all at once.
```

Comment:
~~aren't frameworks, but~~

Trying to avoid giving "framework" a technical meaning independent of target framework